code/modules/admin/verbs/adminjump.dm

```diff
@@ -15,7 +48,7 @@
 			continue
 		if(locate(/obj/structure/grille, T)) // Quick check to not spawn in windows
 			continue
-		turfs.Add(T)
+		turfs += T
 
 	var/turf/T = pick_n_take(turfs)
```

Comment:
```suggestion
	var/turf/T = safepick(turfs)
```
No clue why they `pick_n_take` here... maybe to avoid empty list exceptions? anyway `safepick` does that but cheaper

Reply:
Yeah not sure why it's removing stuff from the list.